misc/review/IMidiEndpointMessageProcessingPlugin.idl

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+// ============================================================================
+// This is part of the Windows MIDI Services App SDK and should be used
+// in your Windows application via an official binary distribution.
+// Further information: https://github.com/microsoft/MIDI/
+// ============================================================================
+
+#include "namespace_defines.h"
+#include "midl_defines.h"
+MIDI_IDL_IMPORT
+
+
+import "MidiMessageReceivedEventArgs.idl";
+import "IMidiEndpointConnectionSource.idl";
+
+
+namespace MIDI_MIDL_NAMESPACE
+{
+    MIDI_API_CONTRACT_v1
+    [MIDI_INTERFACE_UUID("198d5ef2-313f-41af-9259-a42997e060f0",1.0)]
+    interface IMidiEndpointMessageProcessingPlugin
+    {
+        Guid Id { get; };            
+
+        String Name{ get; set; };
```

Comment:
>{ get; set; } [](http://example.com/codeflow?start=19&length=13)

get and set is the default, so our convention is to omit this, (including the curly brackets) in this case.

Reply:
Thanks Anders. I've made this change for the next PR to this repo.